README.md

```diff
@@ -77,6 +77,7 @@ export default defineNuxtConfig({
 #### Options
 
 When you initialize the module, you can pass all [_@storyblok/vue_ options](https://github.com/storyblok/storyblok-vue#storyblok-api) plus a `bridge` option explained in our [JS SDK Storyblok bridge section](https://github.com/storyblok/storyblok-js#storyblok-bridge).
+If you want to use Storyblok inside `nuxt-devtools` you can use the option `devtools`.
```

Comment:
@markus-gx can we add a note on the lines of "if you enable the devtools option, make sure to have installed the `@nuxt/devtools` module and enable it on your nuxt config"

If you feel like you need extra space, we can create a small h3 section for the devtools